scripts/build_twilio_library.py

```diff
@@ -85,6 +86,13 @@ def get_domain_info(oai_spec_location: str, domain: str, is_file: bool = False)
     parts = re.split(r'twilio_(.+?)_?(v\d+)?\.', domain, flags=re.IGNORECASE)
     domain_name = parts[1]
     api_version = parts[2] or ''
+    # special handling for files like twilio_lookups_bulk.json.
+    # This has to be removed when naming is made consistent across all languages
+    if api_version == '':
+        if {'_' in domain_name} and domain_name != DOMAIN_PREVIEW:
```

Comment:
why is second check needed, `domain_name != DOMAIN_PREVIEW` since for `DOMAIN_PREVIEW` it will never pass the first check `{'_' in domain_name}`

Reply:
yes, second check is not required, removed now